backend/src/generated/graphqlgen.ts

```diff
@@ -308,10 +378,6 @@ export namespace MutationResolvers {
 
   export interface PostInput {
     title: string;
-    author: UserInput;
-  }
-  export interface UserInput {
-    name: string;
```

Comment:
you totally deleted the UserInput? why? what was the need/function of this type ?